infra/postdeployment.tf

```diff
@@ -42,9 +42,10 @@ resource "google_compute_instance" "initialize" {
     google_cloud_run_v2_job.client,
   ]
 
-  name         = "head-start-initialize"
-  machine_type = "n1-standard-1"
-  zone         = var.zone
+  name           = "head-start-initialize"
+  machine_type   = "n1-standard-1"
```

Comment:
question: Is there any particular reason why not use [cloud_run_v2_job](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_job) or [google_cloudbuild_trigger](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloudbuild_trigger) instead of GCE?

Reply:
We are using cloud_run_v2_job, see `infra/jobs.tf`, but while TF can ensure these resources exist, TF cannot execute them. The current solution to this issue is the GCE metadata startup script workaround. 

Cloud Build would just either call the jobs themselves (what the metadata startup script currently encapsulates), or break out the jobs into literal Cloud Build (but since we want them separate later for tutorials, this was the simplest solution). Avocano's self-contained TF opts to use Cloud Build.